src/components/TransactionItemRow/index.tsx

```diff
@@ -20,17 +21,24 @@ function TransactionItemRow({
     shouldUseNarrowLayout,
     isSelected,
     shouldShowTooltip,
+    shouldShowChatBubbleComponent = false,
 }: {
     transactionItem: Transaction;
     shouldUseNarrowLayout: boolean;
     isSelected: boolean;
     shouldShowTooltip: boolean;
+    shouldShowChatBubbleComponent?: boolean;
 }) {
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
 
     const backgroundColor = isSelected ? styles.buttonDefaultBG : styles.highlightBG;
     const hasCategoryOrTag = !!transactionItem.category || !!transactionItem.tag;
+    const transactionItemRowRBRStyles = !shouldShowChatBubbleComponent || hasCategoryOrTag ? [styles.ml3, styles.mb3, styles.mtn1] : [];
+    const shouldShowChatBubble = {
+        nextToRBR: shouldShowChatBubbleComponent && !hasCategoryOrTag,
+        nextToCategoryOrTag: shouldShowChatBubbleComponent && hasCategoryOrTag,
```

Comment:
Why do we need this object? Shouldn't we always show it near the end of the row, next to the total?